conjur/cli.py

```diff
@@ -59,143 +63,154 @@ def copyright():
         This method builds the copyright description
         """
         return '''
-Copyright © 2020 CyberArk Software Ltd. All rights reserved.
+Copyright 2020 CyberArk Software Ltd. All rights reserved.
 <www.cyberark.com>
 '''
 
-    # pylint: disable=no-self-use, too-many-locals
+    # pylint: disable=no-self-use, too-many-locals, line-too-long
```

Comment:
This applies the exception to the whole method. Why not do per-line exceptions so that overuse is avoided?